.github/labeler.yml

```diff
@@ -140,7 +140,7 @@
   - "**/attachments/**"
   - "**/**attachment**"
 
-"feature: authentication":
+"feature: authentification":
```

Comment:
`authentication` seems to be the preferable English word: https://english.stackexchange.com/questions/7844/is-authentification-a-real-word

Reply:
yeah but currently we have this everywhere wrong (including the label) 😅 should we rename the label?

Reply:
oh... yeah i guess 😅

but then lets merge this first so that for now there is consistency.